aspnetcore/release-notes/aspnetcore-10/includes/openApi.md

```diff
@@ -77,22 +73,47 @@ In .NET 10 the transformer to do the same task will look like this:
     });
 ```
 
-Note that these changes will be necessary even if you congfigure the OpenAPI version to 3.0.
+The following example shows the changes in diff format.
```

Comment:
@mikekistler  should I delete the before/after code and just have the `diff`? I prefer the `diff`.